draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1626,12 +1633,17 @@ client uses `get-all-by-hash`, then it will already have the new STH.
 
 ### Evaluating compliance
 
-To be considered compliant, a certificate MUST be accompanied by at least one
-valid SCT. A certificate not accompanied by any valid SCTs MUST NOT be
-considered compliant by TLS clients.
+To be considered compliant with this specification, a certificate MUST be
```

Comment:
IIRC, "with this specification" was present in a previous iteration of the document, but we took the decision to remove it because it's implied.